kernels/volk/volk_32f_8u_polarbutterfly_32f.h

```diff
@@ -259,7 +258,6 @@ static inline void volk_32f_8u_polarbutterfly_32f_u_avx(float* llrs,
 
         int p;
         for (p = 0; p < stage_size; p += 8) {
-            _mm256_zeroupper();
```

Comment:
I wonder if this zeroupper call should stay here too. It is part of a loop where SSE and AVX instructions are interleaved.